dired-narrow.el

```diff
@@ -264,7 +264,7 @@ read from minibuffer."
 
 (defun dired-narrow--regexp-filter (filter)
   (condition-case nil
-      (re-search-forward filter (line-end-position) t)
+      (string-match-p filter (buffer-substring (point) (line-end-position)))
```

Comment:
I think instead of relying on `point` it would be better to fetch the file name at line, for example with `(dired-utils-get-filename 'no-dir)`.

Reply:
Done!  Thanks for the suggestion.